storage-proofs/core/benches/drgraph.rs

```diff
@@ -15,7 +15,8 @@ fn drgraph(c: &mut Criterion) {
 
                 b.iter(|| {
                     let mut parents = vec![0; 6];
-                    black_box(graph.parents(2, &mut parents).unwrap());
+                    // parents() never fails, call is_ok() to quieten clippy.
```

Comment:
It's out of scope for this PR, but wouldn't it make sense create a new PR which changes `parents()` to not return a `Result`, but just nothing?

Reply:
I initially thought that (even had that written in the commit log at one stage), also during the remove unwrap work this came up. `parents` is a trait method, I didn't dig into it but assumed the result was used by other implementers of the the trait, otherwise why would it be there in the first place? I can definitely look into this further now you mentioned it though.